src/components/AuthenticatedAppHeader/NotificationsPane.jsx

```diff
@@ -82,43 +101,36 @@ export default function NotificationsPane({
                 >
                   <div style={{ display: 'flex' }}>
                     <Avatar src={shane} variant="circular" />
-                    {senderName !== 'N/A' && (
-                      <Text
-                        style={{ maxWidth: 200, margin: '0 20px' }}
-                      >
-                        {intl.formatMessage(
-                          {
-                            id: 'SENT_YOU_A_COLLABORATION_REQUEST',
-                          },
-                          { senderName },
-                        )}
-                      </Text>
+                    {renderDisplayTextBasedOnMessageType(
+                      currentNotificationSchema,
+                      senderName,
+                      intl,
                     )}
-                    {senderName === 'N/A' &&
-                      !get(notification, 'sender_guid') && (
-                        <Text
-                          style={
-                            { maxWidth: 200, margin: '0 20px' } // probably fine with one or the other check above?
-                          }
-                        >
-                          {intl.formatMessage({
-                            id:
-                              'A_COLLABORATION_WAS_CREATED_ON_YOUR_BEHALF',
-                          })}
-                        </Text>
-                      )}
                   </div>
-                  <Button
-                    onClick={async () => {
-                      setActiveCollaborationNotification(
-                        notification,
-                      );
-                      await markRead(get(notification, 'guid'));
-                      refreshNotifications();
-                    }}
-                    style={{ maxHeight: 40 }}
-                    id="VIEW"
-                  />
+                  {get(
+                    currentNotificationSchema,
+                    'viewInNotificationPane',
```

Comment:
I changed the viewButton back to a boolean in the schema. Since all of the other button stuff is no longer in the schema, and the view button is the only button that appears in the NotificationPane, this feels like a more natural way to handle it in the new setup.

Reply:
Sounds great!

Reply:
could use `currentNotificationSchema?.viewInNotificationPane` here instead if you want

Reply:
It's a little weird to chain `{condition && condition && <component />}` across multiple lines. you could do something like this in the function body instead 

`const showViewButton = currentNotificationSchema?.viewInNotificationPane && refreshNotifications !== undefined`

then in render `{showViewButton && ...`

Reply:
Done